gradle.properties

```diff
@@ -1 +0,0 @@
-org.gradle.jvmargs=-Xmx4g -XX:MaxMetaspaceSize=4g
```

Comment:
I removed in favor of passing the parameter only when needed (the 'check' task). Please tell me if it's needed somewhere else.